scripts/run_integration_tests.sh

```diff
@@ -52,32 +72,48 @@ input_event_files=$(ls ./input_events)
 # Sort event files by name so that snapshots stay consistent
 input_event_files=($(for file_name in ${input_event_files[@]}; do echo $file_name; done | sort))
 
-# Generate a random 8-character ID to avoid collisions with other runs
-run_id=$(xxd -l 4 -c 4 -p < /dev/random)
-
-# Always remove the stack before exiting, no matter what
+# Always remove the stacks before exiting, no matter what
```

Comment:
Nit: I see you changed stack -> stacks here, was that intentional? Isn't this still going to deploy one stack at a time?

Reply:
I guess it should now be `stack(s)`

Reply:
✅